src/obofoundry/utils.py

```diff
@@ -4,15 +4,12 @@
 
 import yaml
 
+from obofoundry.constants import ONTOLOGY_DIRECTORY
+
 __all__ = [
     "get_data",
-    "ONTOLOGY_DIRECTORY",
```

Comment:
What is the significance of this specific removal here?

Reply:
made this a constant that could be reused, therefore it doesn't need to get exported from `utils.py`

Reply:
cool